spyder/utils/programs.py

```diff
@@ -82,42 +82,32 @@ def is_program_installed(basename):
     """
     home = get_home_dir()
     req_paths = []
-    if sys.platform == 'darwin':
-        if basename.endswith('.app') and osp.exists(basename):
-            return basename
-
+    if (
+        sys.platform == 'darwin'
+        and basename.endswith('.app')
+        and osp.exists(basename)
+    ):
+        return basename
+
+    if os.name == 'posix':
         pyenv = [
+            osp.join(home, '.pyenv', 'bin'),
             osp.join('/usr', 'local', 'bin'),
-            osp.join(home, '.pyenv', 'bin')
-        ]
-
-        # Prioritize Anaconda before Miniconda; local before global.
-        a = [osp.join(home, 'opt'), '/opt']
-        b = ['anaconda', 'miniconda', 'anaconda3', 'miniconda3']
-        conda = [osp.join(*p, 'condabin') for p in itertools.product(a, b)]
-
-        req_paths.extend(pyenv + conda)
-
-    elif sys.platform.startswith('linux'):
-        pyenv = [
-            osp.join('/usr', 'local', 'bin'),
-            osp.join(home, '.pyenv', 'bin')
         ]
 
         a = [home, '/opt']
```

Comment:
```suggestion
        a = [home, osp.join(home, 'opt'), '/opt']
```

This is missing from the removed code above and I think it's a possibility on Mac.

Reply:
Okay, I can add that as well. Isn't that a really old standard?

Reply:
I suppose it doesn't hurt, no? It could still be the case if people first installed a conda-based distribution a long time ago, perhaps (since I know many folks who don't reinstall it for years and years)?

Reply:
> Okay, I can add that as well. Isn't that a really old standard?

I've seen people that still use it.

> I suppose it doesn't hurt, no?

Agreed